internal/cmd/cmd.go

```diff
@@ -107,6 +117,42 @@ func apiClient(host string, accessKey string, transport *http.Transport) *api.Cl
 			Timeout:   60 * time.Second,
 			Transport: transport,
 		},
+		OnUnauthorized: logoutCurrent,
```

Comment:
I think we only want to logout in the case that a 401 is returned on `/tokens` for now, otherwise running a command you're not authorized for (ex: `infra users list`) will log you out.

Reply:
I think `OnUnauthorized` is only called when the response code is 401, right?

Reply:
yes. Unauthorized is specific to authn problems, unrelated to authz and access.